src/components/drawer/drawer.js

```diff
@@ -9,6 +9,21 @@ import { tagsModule } from "../tags/tags";
 export class DrawerController {
   constructor($mdSidenav) {
     this.$mdSidenav = $mdSidenav;
+    this.isOpened = false;
+  }
+  addAccountClick(event) {
+    console.log();
```

Comment:
why do you add `addAccountClick` and `addTagClick`  in drawer?

Reply:
this changes are already in master

Reply:
green lines mean you add this code
here is drawer from master
![image](https://user-images.githubusercontent.com/14272396/38331249-6b4ad6c8-385b-11e8-8253-756ee1c8592b.png)